### ISSUE TITLE
Updating GlobeTexture on runtime

### ISSUE THREAD

**Issue by birdm4n** [chrisrzhou/react-globe#50]
Hey! I was wondering if it's in any way possible to update the globe's texture on runtime. 
I've been digging a bit through the source but wasn't able to do this. 
Whenever I change the texture in the prop or through other means, the globe disappears completely. 
Is there no way to do that? Or is that behavior a bug? Thanks for the help!

**Comment by tgmof** [comment]
At least with the black texture globe_dark he provides, it works, isn't it?

**Comment by birdm4n** [comment]
Not really. At least not the way I tried it. Made a small example based on on of the examples: 
https://codesandbox.io/s/react-globe-change-globe-texture-h1vjx

The globe completely goes away when you click the button to change the texture. And I am just using those provided textures there. How did you do it to make it work?
